README.rst

```diff
@@ -875,6 +878,51 @@ These options are supported now:
 
 * ``stateboard-fd-limit`` - ``LimitNOFILE`` option for stateboard instance.
 
+We provide the ability to cache paths for packaged applications. For example, you
+package an application multiple times, and the same rocks are installed each time.
+You can speed up the rebuild process by specifying cached paths in the ``pack-cache.yml`` file.
+By default, we suggest caching the ``.rocks`` directory - we put this path in the standard
+application template.
+
+.. code-block:: yaml
+
+    '.rocks':
+      key-path: myapp-scm-1.rockspec
+
+    'node_modules':
+      always-cache: true
+
+    'third_party/custom_module':
+      key: simple-hash-key
```

Comment:
I was thinking about this option. Is there some way to use this option for a real project? Only if we want manually update the cache key?

Reply:
Yes, it seems only in this case. But I thnik it might be useful